ledger/catchpointtracker.go

```diff
@@ -1178,10 +1198,11 @@ func (ct *catchpointTracker) generateCatchpointData(ctx context.Context, account
 		With("catchpointLabel", catchpointGenerationStats.CatchpointLabel).
 		Infof("Catchpoint data file was generated")
 
```

Comment:
should we add `stateProofVerificationContextHash` to the `catchpointGenerationStats` and log it?

Reply:
on second thought, I'm not sure we should do it.
for example, there is no merkle root hash here, so it might be weird to add `stateProofVerificationContextHash `

what do you think?

Reply:
solution: add ledgerRoundBlockHash, balancesMerkleRoot, encodedTotals and stateProofVerificationContextHash to stats